heareval/model/baseline.py

```diff
@@ -15,9 +14,16 @@
 
 
 class RandomProjectionMelEmbedding(torch.nn.Module):
+    # sample rate, embedding size, and version number are
+    # required model attributes for the HEAR API
+    # TODO how do we want to handle version?
+    sample_rate = 44100
+    embedding_size = 4096
+    version = "0.0.1"
```

Comment:
Let's take out the version number and make an issue for it. The version number I think will be associated with code AND a .pt file, not just code